frontend/ui/widget/opdsbrowser.lua

```diff
@@ -645,6 +660,51 @@ function OPDSBrowser:showDownloads(item)
     UIManager:show(self.download_dialog)
 end
 
+function OPDSBrowser:browse(browse_url, username, password)
+    logger.dbg("Browse opds url", browse_url)
+    table.insert(self.paths, {
+        url = browse_url,
+        username = username,
+        password = password,
+    })
+    if not self:updateCatalog(browse_url, username, password) then
+        table.remove(self.paths)
+    end
+end
+
+function OPDSBrowser:browseSearchable(browse_url, username, password)
+    self.search_server_dialog = InputDialog:new{
+        title = _("Search OPDS catalog"),
+        input = "",
+        hint = _("Search string"),
+        input_hint = _("author:dumas alexandre"),
```

Comment:
I overlooked the fact that this was put in as a translated string, but this `author:` prefix sounds like a technical thing that can't be translated?